hathor/websocket/factory.py

```diff
@@ -54,6 +54,14 @@
     HathorEvents.WALLET_ELEMENT_VOIDED.value
 ]
 
+# Possible channels that can be subscribed on the websocket
+# Each channel has a list of events that emits.
+CHANNELS = {
+    'wallet-service': [
```

Comment:
I'm not sure about using channels. The full node shouldn't have to know this information. It shouldn't need to be aware of the wallet-service, for instance. In the future, if we decide we need an extra message on this service, we'll have to update both the service and the full node.

Ideally, all websocket clients should subscribe to the messages they require. It's their responsibility.

Reply:
The idea for the channels is just to make it easier for people to integrate with the full node ws. If someone wants to make another explorer, doesn't need to know all messages the explorer uses. It can just subscribe to the explorer and check all the messages (maybe this is not useful, since the dev needs to handle the messages in the frontend).

But I agree that we should have the possibility to subscribe to messages. I am just wondering if it's useful to have the channels also, so some common services can integrate easier.

I will add the message subscribe and wait for other inputs to discuss if it's useful to keep the channel subscribe also.

Reply:
> If someone wants to make another explorer, doesn't need to know all messages the explorer uses.

This doesn't make sense to me. If I'm developing a new explorer, I need to know which messages it receives. Also, I might want to have different messages. For instance, I might want to create an explorer that doesn't have the statistics information. If I subscribe to the explorer channel, I'll end up getting them.

> I am just wondering if it's useful to have the channels also, so some common services can integrate easier.

We shouldn't do that, as the full node would need to have knowledge of the other services. This goes agains of what we've been doing, moving logic out of the full node (tx-mining, wallet-service) and leaving the full node only for "essential" tasks.

Reply:
I've updated the code and it's now subscribing to messages. Instead of the channels.

Can you see if it's good?

Reply:
Even though I understand @pedroferreira1's idea, I agree with @obiyankenobi. Anyway, we should have a list of all possible events to subscribe in the documentation of hathor-core.

Reply:
Yes, I agree. Where do you think we should have it? In the README?